app/helpers/organisations_helper.rb

```diff
@@ -4,7 +4,7 @@ module OrganisationsHelper
   OFSTED_REPORT_ENDPOINT = "https://reports.ofsted.gov.uk/oxedu_providers/full/(urn)/".freeze
 
   def age_range(school)
-    return t("schools.not_given") unless school.minimum_age? && school.maximum_age?
+    return I18n.t("vacancies.listing.schools.not_given") unless school.minimum_age? && school.maximum_age?
```

Comment:
just a question really, why you use  I18n here, i thought used that in specs and just t in code?

Reply:
Because we use it in a spec. When this method is called from the spec it raises an error if we don't use I18n.